src/main/java/edu/unc/lib/boxc/migration/cdm/services/ProjectPropertiesService.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright 2008 The University of North Carolina at Chapel Hill
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.unc.lib.boxc.migration.cdm.services;
+
+import edu.unc.lib.boxc.migration.cdm.exceptions.MigrationException;
+import edu.unc.lib.boxc.migration.cdm.model.MigrationProject;
+import edu.unc.lib.boxc.migration.cdm.util.ProjectPropertiesSerialization;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Service for manually updating the project properties
+ * @author krwong
+ */
+public class ProjectPropertiesService {
+
+    private MigrationProject project;
+
+    /**
+     * List project properties that can be manually set/unset
+     * @throws IOException
+     */
+    public void getProjectProperties() throws IOException {
+        Map<String, String> map = new HashMap<String, String>();
+        map.put("hookId", project.getProjectProperties().getHookId());
```

Comment:
Please make constants for the hookId and collectionNumber field names, then use those constants in all the methods in this class.

Also, I wanted to mention that it is possible to construct the names of methods using strings and invoke them programmatically, versus adding cases for each field. This can be done using reflection:
https://www.baeldung.com/java-method-reflection

```
Method getterMethod = MigrationProjectProperties.class.getMethod("getHookId");
```
It would probably be fine to do this in another ticket focused on making more fields listable though since it'd be a pretty significant change.